### DISCUSSION TITLE
Dropping 'SMPTE Timing' as a setting

### DISCUSSION THREAD

**Discussion by chenlung** [SubtitleEdit/subtitleedit#7782] [General]
Similar to what Tero Subtitler did (see [this](https://github.com/URUWorks/TeroSubtitler/discussions/127)), could we rid of the SMPTE setting? Unless I'm missing something, it just causes issues.

I propose the following:
- If the user displays HH:MM:SS:FF, it's SMPTE mode.
- If the user displays HH:MM:SS:MS, it's media mode.

**Comment by niksedk** [comment]
Wauw, what a rip of SE...

**Reply by chenlung** [reply]
?

**Reply by niksedk** [reply]
Seems to be a rip of SE, exactly the same features and components, description fits SE too :(

**Reply by chenlung** [reply]
I'm not here to start an argument, but I don't know if that's true. Could it not be argued that Subtitle Edit may have borrowed from other applications?

Anyway, why close this issue? Seems valid all the same.

**Comment by chenlung** [comment]
SE has potential but also some basic issues, so I'm trying to help. This doesn't seem a very mature response.
